js_src/dart_env_dev.js

```diff
@@ -1,12 +1,5 @@
 React.__isDevelopment = true;
 
-if (typeof window.MemoryInfo == "undefined") {
-  if (typeof window.performance.memory != "undefined") {
-    window.MemoryInfo = function () {};
-    window.MemoryInfo.prototype = window.performance.memory.__proto__;
-  }
-}
-
```

Comment:
I don't think we should remove this until we're also ready to raise the SDK minimum to a version that includes this fix. Otherwise, we'd risk pulling this bug fix out from underneath anyone still using 2.7.

Reply:
That's on me. Nice catch, thank you.